src/19 - Summaries.rsc

```diff
@@ -1304,8 +1305,56 @@ Macro "VMT_Delay Summary" (Args)
             df.rename(name, new_name)
         end
     end
-    df.write_csv(output_dir + "/VMT_Delay_by_" + var +".csv")
+    if Typeof(var) = "string" then df.write_csv(output_dir + "/VMT_Delay_by_" + var +".csv")
```

Comment:
I'm not clear on this. When would var not be a string?

Reply:
On line 1296, I added {"County", "HCMType"} to the group_fields, which is a vector. CAMPO wants the VMT_Delay_by_HCMType_by_County summary back.